GOVERNANCE.md

```diff
@@ -0,0 +1,225 @@
+Such issues must be escalated to the CNCF mediator, Mishi Choudhary <mishi@linux.com>, in which case CNCF may choose to intervene.
+
+## Member Roles
+
+The Flux community is comprised of the following member roles:
+
+### Users
+
+Flux end users are the most important members of the community, without whom the project would have no purpose. Users are anyone who has a need for the project.
+Apart from following the Code of Conduct, there are no special requirements.
+As users continue to engage and become more involved with the Flux community, they may find themselves becoming contributors.
+
+### Contributors
+
+Flux welcomes all kinds of contributions, including code, issues, documentation, external tools, advocacy and community work.
+Contributors are accepted based on the merit of their contributions.
+Contributions help Maintainers ensure that the project continues to satisfy end user needs.
+Teams defined below may have different contribution processes and requirements for the various areas of Flux appropriate to their team.
+
+### Maintainers
+
+Maintainers are elected Contributors who serve on one or more Teams.
+For the election process, see Membership Changes below.
+
+Maintainers are expected to:
+
+- enable and promote Flux community Values
+- engage with end Users through appropriate communication channels appropriate to their teams
+- serve as a point of conflict resolution between Contributors to the Flux area appropriate to their teams
+- maintain open collaboration within and across teams
+- additional responsibilites are defined per team below
+
+## Teams
+
+The Flux project is governed by Maintainer teams, each with defined scope and resonsibilities.
+Teams are responsible for the overall organization, code development, community management, and security.
+
+The goal of governing by teams is to allow parts of the project to act independently as appropriate, enabling the project to grow horizontally while remaining well-maintained.
+
+Teams below are also mapped to GitHub teams within the `fluxcd` GitHub org.
+Their members are mapped to GitHub user accounts.
+
+### Org team
+
+@fluxcd/org <https://github.com/orgs/fluxcd/teams/org>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilies, as well as
+- overseeing the project health and growth
+- maintaining the brand, mission, vision, values, and scope of the overall project
+- changes to licencing and intellectual property
+- administering access to all project assets
+- handling code of conduct violations
+- managing financial decisions
+- defining the scope and responsibilities of all other teams
+- making descisions that affect multiple teams
+- resolving escalated team decisions when the teams responsible are blocked
+- changes to governance (this document)
+
+### Dev team
+
+@fluxcd/dev <https://github.com/orgs/fluxcd/teams/dev>
+
+Scope and responsibilities:
+
+- shared Maintainers responsibilies, as well as
```

Comment:
```suggestion
- shared Maintainers responsibilities, as well as
```

Reply:
I fixed this with a colon instead. Does this read more clearly now?

Reply:
haha "responsibilies". This is pretty funny as far as typos go

Thanks, had to fix in separate commit as opposed to committing your suggestion since I had already pushed changes 👍